pkg/collect/host_system_package.go

```diff
@@ -132,7 +151,7 @@ func (c *CollectHostSystemPackages) Collect(progressChan chan<- interface{}) (ma
 		switch info.OS {
 		case "ubuntu":
 			cmd = exec.Command("dpkg", "-s", p)
-		case "centos", "rhel", "amzn", "ol":
+		case "centos", "rhel", "amzn", "ol", "rocky":
```

Comment:
Since `rocky` and a few others are our friends, I guess we need to reflect this in https://kurl.sh/docs/install-with-kurl/system-requirements#supported-operating-systems. Or is this change just limited to troubleshoot?

Reply:
This change is limited to troubleshoot. There is a larger change coming for kurl and I will include changes to the docs with that.